geps/gep-3539/index.md

```diff
@@ -0,0 +1,200 @@
+# GEP-3539: ClusterIP Gateway - Gateway API to Expose Pods on Cluster-Internal IP Address
+
+* Issue: [#3539](https://github.com/kubernetes-sigs/gateway-api/issues/3539)
+* Status: Memorandum
+
+## TLDR
+
+Gateway API enables advanced traffic routing and can be used to expose a 
+logical set of pods on a single IP address within a cluster. It can be seen 
+as the next generation ClusterIP providing more flexibility and composability 
+than Service API. This comes at the expense of some additional configuration 
+and manageability burden.
+
+## Goals
+
+* Define Gateway API usage to accomplish ClusterIP Service style behavior
+* Propose DNS layout and record format for ClusterIP Gateway
```

Comment:
It doesn't seem like we have fleshed this out. Compared to https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/ we have just 1-2 sentences with a lot of ambiguity here.

Reply:
That is correct, we haven't. DNS bit has been identified as a good candidate to be split out into its own GEP

Reply:
I've expressed elsewhere, but will say again - my ideal outcome is that users could 1-for-1 convert Services (maybe not "while running", but convert YAML) into this new form of Gateway and not change their clients at all.  Ideally this new thing gets the same name as the equivalent Service.